src/client/components/app/Transactions/EnableSpigotTx.tsx

```diff
@@ -197,7 +196,7 @@ export const EnableSpigotTx: FC<EnableSpigotTxProps> = (props) => {
   const renderFuncSelectors = () =>
     revFuncDisplayConfigs.map(({ header, byteCode, options, type, onChange, onByteChange }) =>
       // if no ABI, input bytecode manually
-      selectedContractFunctions ? (
+      isValidAddress(revenueContractAddy) && contractABI ? (
```

Comment:
ah u do the check down here instead and load all functions if we have it at the start?